cmd/limactl/unprotect.go

```diff
@@ -0,0 +1,38 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/lima-vm/lima/pkg/store"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+)
+
+func newUnprotectCommand() *cobra.Command {
+	var unprotectCommand = &cobra.Command{
+		Use:               "unprotect INSTANCE [INSTANCE, ...]",
+		Short:             "Unprotect an instance",
+		Args:              WrapArgsError(cobra.MinimumNArgs(1)),
+		RunE:              unprotectAction,
+		ValidArgsFunction: unprotectBashComplete,
+	}
+	return unprotectCommand
+}
+
+func unprotectAction(_ *cobra.Command, args []string) error {
+	for _, instName := range args {
+		inst, err := store.Inspect(instName)
```

Comment:
Shouldn't we show a warning (not an error) if the instance isn't protected?

```go
if !inst.Protected {
    logurs.warn("Instance %q isn't protected", instName)
    continue
}
```

This might alert the user that they may have specified the wrong instance name, because they clearly expected it to be protected.

Reply:
Fixed